napari/layers/shapes/_tests/test_shapes.py

```diff
@@ -2153,3 +2153,14 @@ def test_world_data_extent():
     max_val = (9, 30, 15)
     extent = np.array((min_val, max_val))
     check_layer_world_data_extent(layer, extent, (3, 1, 1), (10, 20, 5), False)
+
+
+def test_set_data_3d():
+    lines = [
```

Comment:
I would add a link to the issue in a comment, just so we don't delete in the future thinking it redundant!